README.md

```diff
@@ -2,3 +2,7 @@ eclipse-filecompletion
 ======================
 
 File names completion Eclipse plugin
+
+<a href="http://with-eclipse.github.io/" target="_blank">
+<img alt="with-Eclipse logo" src="http://with-eclipse.github.io/with-eclipse-0.jpg" />
```

Comment:
Image looks nice. But not like download image from third site. Is this image used in other plugins ?